### PR TITLE
update package.json and package-lock.json

### PR DESCRIPTION
**Pull request by Kelerchian**

### CODE REVIEW

File: machine-runner/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actyx/machine-runner",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (rkuhn) [inline machine-runner/package.json:3]: let’s call it 0.2.0 — there have been breaking API changes

Comment (Kelerchian) [reply machine-runner/package.json]: Changed to 0.2.0 in https://github.com/Actyx/machines/commit/0cb21a4e64fe127a9c20b9a3389ded703fae4500

**Review by rkuhn — APPROVED** [review]
Yes, let’s release it!

File: dev-example/package-lock.json [hunk 2]
```diff
@@ -44,19 +44,23 @@
       "dev": true,
       "license": "Apache-2.0",
       "devDependencies": {
+        "@actyx/machine-runner": "^0.1.0",
```
[the remark this answers is not in the document]
Comment (Kelerchian) [reply dev-example/package-lock.json:47]: Fixing this in a bit. I need a script to clean build everything anyway

**Pull request merged**